### DISCUSSION TITLE
installing Retronas on a QNAP TS673A NAS

### DISCUSSION THREAD

**Discussion by rester159** [retronas/discussions#33] [Q&A]
Not sure where to start... I can install an Ubuntu Linux station - is that right? Sorry but I'm new to this

**Comment by sairuk** [comment]
start with some kind of linux, follow the [standard installation instructions](https://github.com/danmons/retronas/wiki/Installing-RetroNAS) and see how you go.

Understand we support debian bullseye 64bit on rpi, everything else is up to community effort we don't support ubuntu completely due to a number of reasons (package name changes, missing packages etc) but as we learn about struggles we can look at it.

iirc my QNAP supports VMs but personally have zero interest in running RetroNAS on it, but i'd just approach it as per above.

**Reply by sairuk** [reply]
Took at look at this on my T431P or whatever the model is. Can get debian through "Container Station" and only Debian 10 (buster) is available. The version of Ansible shipped with Debian 10 is too old to use with this project.

this is a far as i got
```
install Container Station
in Container Station install Debian (LXC) (not: only support deb10 32bit which is not compat)

installs to default vol this is dumb

In container station
click install on Debian
click create when ready use a bridge net and add a shared folder to mount at /storage
check overview to see when its running
click vm name
check the console to see root login

apt install sudo python3
cd /usr/bin
ln -s python3 python

useradd -s /bin/bash -m retronas
usermod -a -G sudo retronas
passwd retronas

run the normal retronas installer
```

**Comment by rester159** [comment]
I can probably install a small SSD, install any Linux on it and have it
boot into it. What Linux would you recommend installing? Debian?

**Reply by sairuk** [reply]
we have this [documented](https://github.com/danmons/retronas/wiki/Supported-hardware)
